code/modules/modular_computers/computers/item/role_tablet_presets.dm

```diff
@@ -371,6 +371,12 @@
  * Non-roles
  */
 
+/obj/item/modular_computer/tablet/pda/assistant
```

Comment:
Assistant is a role so shouldn't be in the non-roles section, make a 'no department' section or just put them in service, either is fine.

Reply:
Ding dang done